openspec/changes/add-drift-alert-delivery/design.md

```
@@ -0,0 +1,80 @@
+## Context
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)